<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpinnerMessageWidget</class>
 <widget class="QWidget" name="SpinnerMessageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#progressSpinnerWidget {
    background-image: url(:/assets/images/spinner.png);
    background-repeat : no-repeat;
    background-position:center bottom;
}

#informationText {
	qproperty-alignment: AlignCenter;
	font-family: &quot;Muli&quot;;
	font-weight:&quot;bold&quot;;
	font-size: 24px;
}
</string>
  </property>
  <widget class="QWidget" name="spinnerMessageContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>751</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="spinnerMessageContainer_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="spinnerContainer">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="SpinnerWidget" name="progressSpinnerWidget" native="true"/>
      </item>
      <item>
       <spacer name="horizontalSpacerRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="informationText">
      <property name="text">
       <string>PLaceHolder</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinnerWidget</class>
   <extends>QWidget</extends>
   <header>spinnerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
